UiPharmecy/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$/app/src/main/java/com/example/ahsan/uipharmecy" vcs="Git" />
+  </component>
+</project>
```

Comment:
[Same](https://github.com/AhsanBracu/Image-To-Pdf/pull/1/files#diff-a084b794bc0759e7a6b77810e01874f2R42)